react/src/components/dashboard/dashboard.scss

```diff
@@ -1,3 +1,63 @@
-.dashboard {
+.full-height {
+  height: 100%;
+}
 
+.text-align-center {
+  text-align: center;
+}
+
+.display-none {
+  display: none !important;
+}
+
+.margin-top-30 {
+  margin-top: 30px;
+}
+
+ jumblr.js
+.jumblr-header {
+  font-size: 24px;
+  text-align: center;
+}
+
+// loginModal.js
+.color-white {
+  color: #fff;
+}
+
+//navbar.js
+.padding-top-16 {
+  padding-top: 16px !important;
```

Comment:
please don't use importance param it's going to be quite hard to override it in future